src/documentation/pages/components/forms/search-input/examples/index.vue

```diff
@@ -0,0 +1,32 @@
+<template lang="pug">
+  <Wrapper>
+    h2 Examples
+
+    div(v-if="['lumen', 'centurylink'].includes($store.state.themes.theme)")
+      <Base />
+      <Disabled />
```

Comment:
if these two are common among lumen and portal you can place them above the if conditionals, e.g.:

      <Base />
      <Disabled />
      div(v-if="['lumen', 'centurylink'].includes($store.state.themes.theme)")
          <SizesLumenCenturyLink />